docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -98,15 +98,15 @@ You can create a new business service by following the procedure below.
 ** Owner
 . Click *Create*.
 
-== Creating new tag types
+== Creating new categories
 
-{ProductName} ({ProductShortName}) uses tags in multiple categories and provides a list of default values. You can create a new tag type by following the procedure below.
+{ProductName} ({ProductShortName}) uses tags in multiple categories and provides a list of default values. You can create a new category by following the procedure below.
```

Comment:
can you please replace with " ... You can create a new **tag category** by following the ... "

Reply:
Done